src/actions/meetingActions.js

```diff
@@ -44,10 +44,10 @@ export const createMeeting = info => dispatch => {
     });
 };
 
-export const updateMeeting = info => dispatch => {
+export const updateMeeting = (info, id) => dispatch => {
```

Comment:
please change the order of parameters to have the consistency - so first `id` and then the `object` you want to pass